web/assets/js/main_out.js

```diff
@@ -500,23 +520,26 @@
     var mouseX = NaN;
     var mouseY = NaN;
     var mouseZ = 1;
+    var skinList = [];
 
     var settings = {
-        mobile: "createTouch" in document,
-        showMass: false,
+        nick: "",
```

Comment:
pointless?

Reply:
Not pointless, only the properties that are on the settings object will be read from the saved settings object. That way, if in the future we remove a setting, it won't be read from localStorage.

Reply:
ok